### PR TITLE
voidtools.Everything.Alpha version 1.5.0.1315

### PR DESCRIPTION
**Pull request by salkinxd**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/81994)

### CODE REVIEW

**Comment by salkinxd** [comment]
@microsoft-github-policy-service agree [company="hobedere.com"]

**Comment by salkinxd** [comment]
@microsoft-github-policy-service agree

File: manifests/v/voidtools/Everything/Alpha/1.5.0.1315/voidtools.Everything.Alpha.installer.yaml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-  InstallerSha256: A0CC79D5578F1CC6ABD3340ABDE04A0E7FED8AE19A1C090A373A32C2CB0D2737
-  ProductCode: Everything 1.5a
-ManifestType: installer
-ManifestVersion: 1.1.0
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: voidtools.Everything.Alpha
+PackageVersion: 1.5.0.1315
```
Comment (OfficialEsco) [inline manifests/v/voidtools/Everything/Alpha/1.5.0.1315/voidtools.Everything.Alpha.installer.yaml:5]:
```suggestion
PackageVersion: 1.5.0.1321
```

File: manifests/v/voidtools/Everything/Alpha/1.5.0.1315/voidtools.Everything.Alpha.locale.en-US.yaml [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
-- indexing
-- search
-ManifestType: defaultLocale
-ManifestVersion: 1.1.0
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: voidtools.Everything.Alpha
+PackageVersion: 1.5.0.1315
```
Comment (OfficialEsco) [inline manifests/v/voidtools/Everything/Alpha/1.5.0.1315/voidtools.Everything.Alpha.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.5.0.1321
```

File: manifests/v/voidtools/Everything/Alpha/1.5.0.1315/voidtools.Everything.Alpha.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-PackageVersion: 1.5.0.1315
-DefaultLocale: en-US
-ManifestType: version
-ManifestVersion: 1.1.0
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: voidtools.Everything.Alpha
+PackageVersion: 1.5.0.1315
```
Comment (OfficialEsco) [inline manifests/v/voidtools/Everything/Alpha/1.5.0.1315/voidtools.Everything.Alpha.yaml:5]:
```suggestion
PackageVersion: 1.5.0.1321
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Since 1.5.0.1315 is still available you should probably post a new version instead of deleting the previous version

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by PR 82819 to avoid pull request being stale;

**Pull request closed without merging**